src/auth/components/SelectTenant.jsx

```diff
@@ -60,6 +65,17 @@ export default class SelectTenant extends Component {
                 defaultValue={''}>
                 {this.buildSelectOptions()}
               </select>
+              <div className="form-group">
+                <div className="checkbox enable-tenant-filter">
+                  <label>
+                    <input type="checkbox" onChange={this.handleFilterTenantStatusChange}
+                      checked={this.state.tenantFilter}
+                    />
+                    Filter resources by tenant
+                  </label>
+                  <span className="help-block">( Modifiable after login too )</span>
```

Comment:
Please add space before `(` and remove spaces after `(` and before `)`, and change `M` to `m` 🙂